source/ways-of-working/new-component/github-repo.html.md.erb

```diff
@@ -0,0 +1,43 @@
+---
+title: Create Github Repository
+last_reviewed_on: 2020-03-06
+review_in: 6 months
+weight: 3
+---
+
+# <%= current_page.data.title %>
+
+Any developer added to [hmcts org](https://github.com/hmcts) would be able to create a new Github repo.
+
+
+1. Naming convention for repository is `{product}-{component}`.
+2. Choose from a template in the drop down depending on whether it's a front end or backend component. These templates provide all the boiler plate code needed for a new app. 
+    - [spring-boot-template](https://github.com/hmcts/spring-boot-template)
+    - [expressjs-template](https://github.com/hmcts/expressjs-template)
+3. Repository should be public.
```

Comment:
feels like we need a link to coding in the open? or maybe the gds guide to explain this?
i.e. you shouldn't create a private repo unless you really know what you're doing